File: cases/taint/php/argument-position-negative/argument_position_negative.php
<?php
function dfb_source(): string { // DFB-SOURCE: argument-position-negative-input
    return "tainted";
}

function chooseFirst(string $first, string $second): string { // DFB-WITNESS: argument-position-negative-first
    return $first;
}

function dfb_sink(string $value): void {} // DFB-SINK: argument-position-negative-sink

function run(): void {
    $result = chooseFirst("clean", dfb_source());
    dfb_sink($result);
}

File: cases/taint/rust/argument-position-negative/argument_position_negative.rs
fn dfb_source() -> i32 { // DFB-SOURCE: argument-position-negative-input
    1
}

fn choose_first(first: i32, second: i32) -> i32 { // DFB-WITNESS: argument-position-negative-first
    first
}

fn dfb_sink(value: i32) {} // DFB-SINK: argument-position-negative-sink

fn run() {
    let result = choose_first(0, dfb_source());
    dfb_sink(result);
}
